exporting/read_config.c

```diff
@@ -376,15 +385,28 @@ struct engine *read_exporting_config()
             struct aws_kinesis_specific_config *connector_specific_config =
                 callocz(1, sizeof(struct aws_kinesis_specific_config));
 
+            default_destination = "us-east-1";
+
             tmp_instance->config.connector_specific_config = connector_specific_config;
 
             connector_specific_config->stream_name = strdupz(exporter_get(instance_name, "stream name", "netdata"));
-
             connector_specific_config->auth_key_id = strdupz(exporter_get(instance_name, "aws_access_key_id", ""));
-
             connector_specific_config->secure_key = strdupz(exporter_get(instance_name, "aws_secret_access_key", ""));
         }
 
+        if (tmp_instance->config.type == EXPORTING_CONNECTOR_TYPE_PUBSUB) {
```

Comment:
Are we  missing a `#ifndef ENABLE_EXPORTING_PUBSUB` before this `if`?  Case we are not, why do we have this instruction at line `316`?

Reply:
At L316, we are checking if Pub/Sub support is compiled, and if it's not, then L397 will never be executed. No need for an additional `#ifdef`.

Reply:
Case it is not supported, will we have `EXPORTING_CONNECTOR_TYPE_PUBSUB` defined in the default headers? This was the main motive I asked you.

Reply:
Yes, we will have it defined.